package.json

```diff
@@ -1,9 +1,12 @@
 {
   "scripts": {
     "test": "lerna run test",
-    "postinstall": "lerna bootstrap"
+    "postinstall": "lerna bootstrap && link-parent-bin"
   },
   "devDependencies": {
-    "lerna": "2.0.0-beta.30"
+    "ava": "^0.22.0",
+    "lerna": "2.0.0-beta.30",
```

Comment:
Mind upgrading `lerna` to the latest?

Reply:
I think there was an issue with newer versions that prevented me from upgrading in the past. But yeah, we should have a look at that 😉